### PR TITLE
uci

### PR DESCRIPTION
**Pull request by Akababa**
added simple uci support for playing with it in a gui

### CODE REVIEW

**Comment by francescobianco** [comment]
Amazing @Akababa

File: requirements.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@ qtconsole==4.3.1
 scipy==0.19.1
 simplegeneric==0.8.1
 six==1.11.0
-tensorflow==1.3.0
+tensorflow-gpu==1.3.0
```
Comment (ddugovic) [inline requirements.txt]: Can non-GPU users still use `tensorflow` instead?

Comment (Akababa) [reply requirements.txt]: Yeah, sorry I didn't mean to commit that part

**Comment by Akababa** [comment]
@Zeta36 review please?

**Comment by Akababa** [comment]
??

**Pull request closed without merging**